### PR TITLE
Create NewFile1.txt

### PR DESCRIPTION
**Pull request by zhaolih**
please review my new created file NewFile1.txt

### CODE REVIEW

**Review by zhaolihG — COMMENTED** [review]
Below are my comments:
1.sfadsf
2.fasdfdsf
3.fadsfs
4.hteryrwetg

another comment appended:
5.tertewrq

File: NewFile1.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The IFS_JOB_INFO table function returns a table that contains information about IFS references for a job.
+
+This information is similar to what is returned by the Retrieve Referenced Objects (QP0LRRO) API.
+
+The list of objects returned may be incomplete for objects residing in file systems other than the root (/), QOpenSys, and user-defined file systems. Objects in some of the other file systems can be locked with interfaces that do not use the integrated file system. Therefore, objects referenced by a job will only have references that were obtained as part of an integrated file system operation, or an operation that causes an integrated file system operation to occur.
```
Comment (zhaolih) [inline NewFile1.txt:5]: this is wrong

**Review by zhaolihG — CHANGES_REQUESTED** [review]
change from 'Comment' to 'Request changes'

**Comment by zhaolih** [comment]
fixed per all comments

**Pull request merged**